modules/manage/pages/iceberg/iceberg-topics-gcp-biglake.adoc

```diff
@@ -0,0 +1,272 @@
+= Query Iceberg Topics using GCP BigLake
+:description: Add Redpanda topics as Iceberg tables to your Google BigLake data lakehouse that you can query from Google BigQuery.
+:page-categories: Iceberg, Tiered Storage, Management, High Availability, Data Replication, Integration
+
+[NOTE]
+====
+include::shared:partial$enterprise-license.adoc[]
+====
+
+// tag::single-source[]
+ifndef::env-cloud[]
+:rp_version: 25.3
+:rpk_install_doc: get-started:rpk-install.adoc
+endif::[]
+
+This guide walks you through querying Redpanda topics as Iceberg tables stored in Google Cloud Storage, using a REST catalog integration with https://cloud.google.com/biglake/docs/introduction[Google BigLake^]. You create a new storage bucket to store Iceberg data, create a new service account with required permissions, deploy Redpanda using Docker on Google Cloud VMs, and enable the Iceberg integration for a pre-existing Redpanda topic so you can query it from BigQuery.
+
+For general information about Iceberg catalog integrations in Redpanda, see xref:manage:iceberg/use-iceberg-catalogs.adoc[].
+
+NOTE: Check the https://cloud.google.com/biglake[BigLake product page^] for the latest status and availability of the REST Catalog API.
+
+== Prerequisites
+
+* A Google Cloud Platform (GCP) project.
+* The https://docs.cloud.google.com/sdk/docs/install[`gcloud` CLI^] installed and configured for your GCP project.
+** Your Redpanda cluster must be deployed on GCP VMs. If you do not have permissions to create VMs and service accounts in your project, ask your administrator to create them for you.
+* https://cloud.google.com/biglake/docs/enable-biglake-api[BigLake API^] enabled for your GCP project.
+
+== Limitations
+
+=== Multi-region bucket support
+
+BigLake metastore does not support multi-region buckets. Use single-region buckets to store your Iceberg topics.
+
+=== Catalog deletion
+
+Currently, it is not possible to delete non-empty BigLake Iceberg catalogs through the BigLake interface. If you need to reconfigure your setup, create a new bucket or use the REST API to remove the existing catalog.
+
+=== Topic names
+
+BigLake does not support Iceberg table names that contain dots (`.`). When creating Iceberg topics in Redpanda that you plan to access through BigLake, either:
+
+- Use the `iceberg_topic_name_dot_replacement` cluster property to set a replacement string for dots in topic names. Ensure that the replacement value does not cause table name collisions. For example, `current.orders` and `current_orders` would both map to the same table name if you set the replacement to an underscore (`_`).
+- Ensure that the new topic names do not include dots.
+
+You must also set the `iceberg_dlq_table_suffix` property to a value that does not include dots or tildes (`~`). See <<configure-topic-for-iceberg>> for the list of cluster properties to set when enabling the BigLake REST catalog integration.
```

Comment:
FYI @kbatuigas this property is undocumented in the cluster config reference (when I checked yesterday)